src/resources/js/components/Presentation.vue

```diff
@@ -64,15 +63,8 @@ export default {
     };
   },
   beforeMount() {
-    // set the bearer token
-    window.axios.defaults.headers.common = { Authorization: `Bearer ${localStorage.getItem('Authorization-token')}` };
-
-    /** *************************************************************************
-     * there is an issue with the route here, it appears that route
-     * with the following pattern "aaaa/bbbb" lead to a 404 error
-     */
+ 
```

Comment:
there is a linting issue after the merge, there is an extra line space here

Reply:
fixed